gcp/api/server.py

```diff
@@ -795,6 +800,7 @@ def query_by_version(context: QueryContext,
                      version,
                      to_response: Callable = bug_to_response):
   """Query by (fuzzy) version."""
+  validate_package_name(package_name)
```

Comment:
https://github.com/google/osv.dev/blob/6246f3be6eaaee14f6616171ef7d133c4b3cc68a/gcp/api/server.py#L445 is probably a better place to do this validation, as it is

a) earlier in the call chain
b) will also catch it for `query_by_package()`, which needs this as well